fix: Filepicker final review (#1017)
appsmithorg/appsmith-docs: website/docs/reference/widgets/filepicker.md

:::info
* Any file exceeding 5 MB would be saved as a blob URL, and the upper limit for file size is 100 MB.

> * When using the data in an API or query, it would be uploaded in the selected format, despite appearing in the blob URL format when logged.

Reviewer: API and query are the same

```suggestion
* When using the data in a query, it is uploaded in the selected format, despite appearing in the blob URL format when you log the data.
```

These properties are present in the property pane of the widget. The following table lists all the widget properties.

> | Property                      	| Data Type       	| Description                                                                                                                                                                                                                                                                           	|

Reviewer: no need of code blocks for datatypes. Also add data type column for reference and style properties
